features/features.go

```diff
@@ -97,6 +97,10 @@ type Config struct {
 	//    renewal.
 	//
 	// TODO(#7511): Remove this feature flag.
+	//
+	// Deprecated: This feature flag is deprecated and will be removed in the
```

Comment:
We generally mark flags as deprecated by moving them to the block at the top of the file (line 30).